<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGuiCoreWidget</class>
 <widget class="QWidget" name="SettingsGuiCoreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="_groupBoxMdi">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="_rbtnWindows">
        <property name="text">
         <string>Windows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_rbtnTabs">
        <property name="text">
         <string>Tabs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_checkBoxHScrollbar">
        <property name="text">
         <string>Horizontal scrollbar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_checkBoxVScrollbar">
        <property name="text">
         <string>Vertical scrollbar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="_checkBoxOpenWidgets">
        <property name="text">
         <string>Reopen windows after restart</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="_checkBoxToolButtonTextVisible">
        <property name="text">
         <string>Show tool button text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_checkBoxStatusBarVisible">
        <property name="text">
         <string>Show status bar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
